pkg/scheduler/core/generic_scheduler.go

```diff
@@ -677,6 +677,7 @@ func PrioritizeNodes(
 	nodes []*v1.Node,
 	extenders []algorithm.SchedulerExtender,
 ) (schedulerapi.HostPriorityList, error) {
+	trace := utiltrace.New(fmt.Sprintf("Prioritizing %s/%s", pod.Namespace, pod.Name))
```

Comment:
It looks like you're tracing the steps, but never logging the trace. [:smile:](https://www.youtube.com/watch?v=A7uvttu8ct0)

Since you're debugging performance, I would actually recommend building scheduler with your changes and running that against your cluster. I am in favor of better logging, but you don't have to wait for LGTM to get value out of this PR.

Reply:
Thanks for the reminder - I have added the missing call.